asistente_ladm_col/logic/quality/logic_quality_rules.py

```diff
@@ -0,0 +1,494 @@
+                                                            QUALITY_RULE_ERROR_CODE_E401002,
+                                                            QUALITY_RULE_ERROR_CODE_E401003,
+                                                            QUALITY_RULE_ERROR_CODE_E401004,
+                                                            QUALITY_RULE_ERROR_CODE_E401005,
+                                                            QUALITY_RULE_ERROR_CODE_E401006,
+                                                            QUALITY_RULE_ERROR_CODE_E401007,
+                                                            QUALITY_RULE_ERROR_CODE_E401008,
+                                                            QUALITY_RULE_ERROR_CODE_E401101,
+                                                            QUALITY_RULE_ERROR_CODE_E401102,
+                                                            QUALITY_RULE_ERROR_CODE_E401103,
+                                                            QUALITY_RULE_ERROR_CODE_E401104,
+                                                            QUALITY_RULE_ERROR_CODE_E401105,
+                                                            QUALITY_RULE_ERROR_CODE_E401106,
+                                                            QUALITY_RULE_ERROR_CODE_E401107,
+                                                            QUALITY_RULE_ERROR_CODE_E401108,
+                                                            QUALITY_RULE_ERROR_CODE_E401109,
+                                                            QUALITY_RULE_ERROR_CODE_E401110,
+                                                            QUALITY_RULE_ERROR_CODE_E401111)
+from asistente_ladm_col.config.enums import EnumQualityRule
+from asistente_ladm_col.config.ladm_names import LADMNames
+from asistente_ladm_col.lib.logger import Logger
+from asistente_ladm_col.lib.quality_rule.quality_rule_manager import QualityRuleManager
+from asistente_ladm_col.utils.utils import get_uuid_dict
+
+
+class LogicQualityRules:
+    def __init__(self, qgis_utils):
+        self.quality_rules_manager = QualityRuleManager()
+        self.qgis_utils = qgis_utils
+        self.logger = Logger()
+        self._ladm_queries = dict()
+
+    def get_ladm_queries(self, engine):
+        if engine not in self._ladm_queries:
+            self._ladm_queries[engine] = ConfigDBsSupported().get_db_factory(engine).get_ladm_queries(self.qgis_utils)
+
+        return self._ladm_queries[engine]
+
+    def check_parcel_right_relationship(self, db):
+        rule = self.quality_rules_manager.get_quality_rule(EnumQualityRule.Logic.PARCEL_RIGHT_RELATIONSHIP)
+        error_layer = None
+        error_layer_exist = False
+        group = self.qgis_utils.get_error_layers_group()
+        layers = group.findLayers()
+        for layer in layers:
+            if layer.name() == rule.error_table_name:
+                error_layer = layer.layer()
+                error_layer_exist = True
```

Comment:
We should avoid reusing existing layers for quality validations. This code snippet was used before, but nowadays it does not have a real functionality. Please remove.

Reply:
Yes, It is right.
Thank you, resolved in e9e311a